Formula/rpm.rb

```diff
@@ -19,8 +19,12 @@ class Rpm < Formula
     sha256 x86_64_linux: "1147c07948c53779fdf751623b349d6da6d6b4753103ce2c898da2cc68a0a041"
   end
 
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
   depends_on "berkeley-db"
   depends_on "gettext"
+  depends_on "gnupg"
```

Comment:
Can we pass `"__GPG=#{Formula["gpg"].opt_bin}/gpg"` to `configure` to avoid the extra dependency?

Reply:
Sure thing- I'll defer to you here. Tested that change with and without `gnupg` installed, and the `lib/rpm/macros` file has the GPG path set properly in both cases.